charts/server/Chart.yaml

```diff
@@ -7,8 +7,9 @@ version: 0.1.0
 appVersion: main
 
 dependencies:
-  - name: consul
```

Comment:
I was expecting a move to consul off by default here, till we remove it later?

Reply:
I think we resolved this offline? The chart now defaults to the CRD store so I don't think we should keep this.